addons/setVehicleAmmo/functions/fnc_attributeLoad.sqf

```diff
@@ -176,6 +189,7 @@ private _addWeaponSystem = {
         private _weapon = _x;
         private _muzzles = getArray (configFile >> "CfgWeapons" >> _weapon >> "muzzles");
         private _weaponMagazines = getArray (configFile >> "CfgWeapons" >> _weapon >> "magazines");
+        _weaponMagazines append ([_weapon] call _fnc_gatherMagazineWells);
```

Comment:
this seems to work, but either way is fine
```suggestion
        private _weaponMagazines = compatibleMagazines _weapon;
```

Reply:
Sounds good, changed that and removed the function